src/Connections/FeedConnection.ts

```diff
@@ -10,16 +10,32 @@ import axios from "axios";
 import markdown from "markdown-it";
 import { Connection, ProvisionConnectionOpts } from "./IConnection";
 import { GetConnectionsResponseItem } from "../provisioning/api";
+import QuickLRU from "@alloc/quick-lru";
 
 const log = new LogWrapper("FeedConnection");
 const md = new markdown();
 
+export interface LastResultOk {
+    timestamp: number;
+    ok: true;
+}
+export interface LastResultFail {
+    timestamp: number;
+    ok: false;
+    error?: string;
+}
+
+
 export interface FeedConnectionState extends IConnectionState {
     url:    string;
     label?: string;
 }
 
-export type FeedResponseItem = GetConnectionsResponseItem<FeedConnectionState, object>;
+export interface FeedConnectionSecrets {
```

Comment:
Why is that part of secrets?

Reply:
I didn't want to expose the status of the feed to everyone in the room, but just room admins. We can rethink that logic though, wdyt?

Reply:
Oh, I thought of secrets as "visible to no one", but if they're admin-visible then that makes sense.